src/runtime/runtime.js

```diff
@@ -717,7 +717,21 @@ export class ConfiguredRuntime {
   getEntitlements(encryptedDocumentKey) {
     return this.entitlementsManager_
       .getEntitlements(encryptedDocumentKey)
-      .then(entitlements => entitlements.clone());
+      .then(entitlements => {
+        // Auto update internal things tracking the user's current SKU.
+        if (entitlements) {
+          try {
+            let skus = '';
```

Comment:
How about:
const skus = entitlements.entitlements.map(entitlement => entitlement.getSku());
if (skus.length > 0) {
  this.analyticsService_.setSku(skus.join(','));
}

Reply:
+1

Reply:
Done